src/main/java/org/team1540/elmo/subsystems/ejectors/EjectInnerCommand.java

```diff
@@ -0,0 +1,19 @@
+package org.team1540.elmo.subsystems.ejectors;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import org.team1540.elmo.utils.WaitDashboardVarCommand;
+
+public class EjectInnerCommand extends SequentialCommandGroup {
+
+    public EjectInnerCommand(Ejectors.InnerEjector innerEjector) {
+        addRequirements(innerEjector);
+
+        addCommands(
+                new InstantCommand(innerEjector::extend),
+                new WaitDashboardVarCommand(this,"push",0.15), // wait for suction to fully unsucc. // TODO: Test without waiting
+                new InstantCommand(()->innerEjector.setSuction(false))
```

Comment:
Probably want it to retract the arms at the end too (applies to all 3)

Reply:
ive done this as the default command for the ejectors